velox/functions/sparksql/aggregates/Register.cpp

```diff
@@ -17,12 +17,13 @@
 #include "velox/functions/sparksql/aggregates/Register.h"
 
 #include "velox/functions/sparksql/aggregates/BitwiseXorAggregate.h"
-#include "velox/functions/sparksql/aggregates/LastAggregate.h"
 
 namespace facebook::velox::functions::sparksql::aggregates {
 
+extern void registerFirstLastAggregate(const std::string& prefix);
+
 void registerAggregateFunctions(const std::string& prefix) {
-  aggregates::registerLastAggregate(prefix + "last");
+  registerFirstLastAggregate(prefix);
```

Comment:
nit: since this method registers multiple functions, perhaps, rename to registerFirstLastAggregates with 's' at the end